From fee0ce8fa7992474ab3138ac1c73e3fc83de4fd1 Mon Sep 17 00:00:00 2001
From: Chetan Rao <chetan.rao@intel.com>
Date: Mon, 9 Dec 2013 16:34:15 +0200
Subject: [PATCH] [PATCH] libffi: Enable build for x86

BZ: 157623

Build for libffi library is disabled. Should be enabled as its required by
glib library.

Change-Id: Ie32c374686be4ce4bd01bce392f3164642e0b275
Category: aosp improvement
Domain: AOSP-Framework-CoreService
Origin: internal
Upstream-Candidate: no, proprietary
Signed-off-by: Chetan Rao <chetan.rao@intel.com>
---
 Android.mk |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Android.mk b/Android.mk
index e7c57dd..06b21c2 100644
--- a/Android.mk
+++ b/Android.mk
@@ -23,8 +23,8 @@ LOCAL_PATH:= $(call my-dir)
 # Build rules for the target.
 #
 
-# We only build ffi for mips.
-ifeq ($(TARGET_ARCH),mips)
+# We only build ffi for mips/x86.
+ifneq (,$(filter $(TARGET_ARCH), mips x86))
 
    include $(CLEAR_VARS)
 
-- 
1.7.10.4

